servant/server/src/Mu/Servant/Server.hs

```diff
@@ -445,7 +478,8 @@ sourceToSource (SourceT src) = ConduitT (PipeM (liftIO $ src (pure . go)) >>=)
 -- 2. HTTP method which must be used,
 -- 3. HTTP status code of a successful HTTP response from a specific `Method`. Use 200 for the usual status code.
 data ServantRoute
-  = ServantTopLevelRoute [Symbol]
+  = ServantAdditional (Maybe Type)
```

Comment:
Instead of using `Maybe Type` with `'Nothing` as the identity API, what about just using `Type` with `EmptyAPI` as the identity?

Reply:
Great idea! I've implemented it in the last commit.